docs/en/user_guides/useful_tools.md

```diff
@@ -1,19 +1,19 @@
-## Useful tools
+# Useful tools
 
 We provide lots of useful tools under `tools/` directory.
 
-### Get the FLOPs and params (experimental)
+## Get the FLOPs and params 
```

Comment:
I'm not sure whether params equal to parameter amount. Maybe we can introduce what is ``FLOPs`` and ``params`` and why we need these information for starters.